bin/ldapmanager

```diff
@@ -70,6 +92,41 @@ def main():
     user.add_argument("--mail", default=None, help=(
         "If no mail address is iset, login and base will be used"))
 
+    # Passwd ################################################################
+    passwd = subparsers.add_parser(
+        "passwd", help="reset or add passwords for users",
+        description="""
+Each user can have one or more passwords. This command allows either replacing
+all current passwords (default) or appending passwords with --append. Passwords
+can be set for one user at a time. If no option is chosen, then the current user
+(--bind-user) will be used. --admin will use the value of $LDAP_ADMIN_DN
+(default: uid=admin,ou=system) while --login will change the password of a user
```

Comment:
``--login will change the password....`` -> ``--login  can be used to specify a user in ou=Users``